src/utils/configfile.cc

```diff
@@ -42,7 +42,7 @@ const OptionId kConfigFileId{
     "Path to a configuration file. The format of the file is one command line "
     "parameter per line, e.g.:\n--weights=/path/to/weights",
     'c'};
-const char* kDefaultConfigFile = "lc0.config";
+const char* kDefaultConfigFile = "<default>";
```

Comment:
Sorry for nitpicking, but I suggest moving both constants here:
```cpp
const char* kDefaultConfigFile = "lc0.config";
const char* kDefaultConfigFileParam = "<default>";
```
(or if it's one, better it be the filename and not the option value)

Reply:
Not a problem. Do these changes match what you were thinking?

Reply:
Looks good!